pub.go

```diff
@@ -13,6 +13,11 @@ import (
 	"golang.org/x/xerrors"
 )
 
+// Topics interface
+type Topics interface {
+	Topics() []string
```

Comment:
perhaps document here whether the returned list of topics is expected to be sorted?
(I think for testing purposes, we could indeed document it's sorted. see _e.g._ [sql.Drivers](https://godoc.org/database/sql#Drivers))